articlequality/extractors/ptwiki.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import re
+import traceback
+
+from .extractor import TemplateExtractor
+
+logger = logging.getLogger(__name__)
+
+POSSIBLE_LABELS = ("1", "2", "3", "4", "5", "6")
+
+
+def from_template(template):
+
+    if template.has_param('class'):
+        template_name = normalize_template_name(template.name)
+        project_name = normalize_project_name(template.name)
+        try:
+            label = str(template.get('class').value.strip_code())
+            label = label.strip().lower()
+
+            if label in POSSIBLE_LABELS:
+                return project_name, label
+            else:
+                logger.debug("Class '{0}' not in possible classes."
+                              .format(label))
+                pass # not a quality assessment class
+        except ValueError as e:
+            logger.warning(traceback.format_exc())
+            pass # no assessment class in template
+
+
+TRANSLATIONS = {}
+
+
+def normalize_template_name(template_name):
+    template_name = str(template_name).lower().replace("_", " ")
+    if str(template_name) in TRANSLATIONS:
+        return TRANSLATIONS[template_name]
+    else:
+        return template_name
+
+
+WP_PREFIX = re.compile("{{Marco de projecto| }} ?", re.I)
```

Comment:
Not sure what is going on here, but I don't think we need this.

Reply:
Can you elaborate please, I am unsure what you are talking about here